.rubocop.yml

```diff
@@ -1,8 +1,10 @@
 AllCops:
-  TargetChefVersion: 15.latest
+  TargetChefVersion: 16.latest
   Exclude:
     - 'cookbooks/**/*'
 ChefModernize/FoodcriticComments:
   Enabled: true
 ChefStyle/CopyrightCommentFormat:
   Enabled: true
+Style/RegexpLiteral:
+  EnforcedStyle: mixed
```

Comment:
What is this needed for?

Reply:
The regex in the spec tests uses %r in 2 cases because they contain a / in them but all 3 cases are multi-line, so the %r context fits better.

Reply:
When I used %r on the third multi-line matcher it threw an error, but it looks much cleaner this way

Reply:
I think the mixed behavior makes the most sense because in most cases `/aaadsa/` looks good for single line but 

%r(dasdas
    asdasdas{
      asdsad
  })

vs

/dasdas
    asdasdas{
      asdsad
  }/
looks better for multi-line imo

Reply:
I'd prefer to disable this on a per line basis and not globally if possible.

Reply:
Are you sure? I think this change only affects multi-line regex

Reply:
The mixed keeps // on flat and %r() on multi-line